### DISCUSSION TITLE
Feedback on empty states

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#1054] [General] [answered]
We want feedback on the proposal for a new parttern for empy states pages:  
I'm working on a project that needs two different type of pages: 
1. With no results found and call-to-actions
2. One on empty state, telling the customer the next steps to follow (one main call to action)

[This is the link to the mockups](https://share.goabstract.com/629017ea-ad90-4d5a-8bdf-67373d81f32a).

**Comment by johannaweintraub** [comment, marked as answer]
Not sure if you've seen this - there is a template for pattern contribution if you're able to take that on...https://paste-git-feat-website-add-patterns-poc.twilio-dsys.vercel.app/patterns/empty-state

Other use cases may include

1. one or more buttons used for CTAs
2. Dependency state where you need to do something else before you can create a thing

Here is a thread I had with Katie regarding empty state variations and illustrations fyi. https://twilio.slack.com/archives/CBEQHHDLP/p1608074449211400

**Reply by corinne-nagel** [reply]
Thanks for your reply @johannaweintraub !

**Comment by johannaweintraub** [comment]
Sure thing - I'm going to work with @lgalloherran after the break to get this pattern kicked off.
